packages/devtools_app/lib/src/screens/inspector_v2/layout_explorer/ui/utils.dart

```diff
@@ -441,3 +563,47 @@ class LayoutExplorerBackground extends StatelessWidget {
     );
   }
 }
+
+/// Builds and positions a label for the [LayoutExplorerBackground] as
+/// determined by the widget's padding.
+class PositionedBackgroundLabel extends StatelessWidget {
+  const PositionedBackgroundLabel({
+    super.key,
+    required this.labelText,
+    required this.labelColor,
+    required this.topPadding,
+    required this.bottomPadding,
+    required this.leftPadding,
+    required this.rightPadding,
+  });
+
+  final String labelText;
+  final Color labelColor;
+  final bool topPadding;
+  final bool bottomPadding;
+  final bool leftPadding;
+  final bool rightPadding;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        // Push to the bottom if there is no padding on the top.
+        if (!topPadding && bottomPadding) const Spacer(),
```

Comment:
Can `topPadding` and `bottomPadding` both be true? If so, what's the expected behavior in that case? If not, consider adding assertions to ensure that only one is specified.

Reply:
If both are true, no spacer is needed because the label will stay at the top (instead of being pushed to the bottom with the `Spacer`)